satpy/Combined_colormaps.ipynb

```diff
@@ -0,0 +1,540 @@
+{
```

Comment:
<p>This images doesn't show in reviewnb, is it set up correctly ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pytroll/pytroll-examples/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>